### ISSUE TITLE
[Feature]: Search

### ISSUE THREAD

**Issue by AndreaKarlsen** [HelseVestIKT/hviktor-blazor#111]
### 📝 Problem Statement

~~1. The input is missing form~~  
~~Applies to all examples. A search should always be coded as a form.~~

2. The ~~input~~ form has the wrong role
3. The input type should be search

~~4. The aria-label should be in the same language as the page (english)The aria-label is "Søk", but all other content is in english. Should be "Search"~~

~~5. Label is not associated with input~~  
~~For the examples with label they are not connected to input~~

~~6. Form example is missing type as searchHowever, I do not see the use of a variant with "submitted value" and we could probably remove this example all together.~~

**Comment by Cusatelli** [comment]
I will investigate if the input has the wrong role logic in the component or if it is an error in the documentation only.

Similarly I will ensure the aria-label is correctly set in the component logic as well as linking labels to the input field.

**Comment by Cusatelli** [comment]
In this issue I will resolve the incorrect input type value and set it to "search" as described. See #116 for localization support changes.
